packages/CodingStandard/src/Fixer/Property/ArrayPropertyDefaultValueFixer.php

```diff
@@ -0,0 +1,77 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Fixer\Property;
+
+use Nette\Utils\Strings;
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+
+final class ArrayPropertyDefaultValueFixer extends AbstractFixer
+{
+    public function isCandidate(Tokens $tokens): bool
+    {
+        // analyze only properties with comments
+        return $tokens->isAllTokenKindsFound([T_DOC_COMMENT, T_VARIABLE]);
+    }
+
+    protected function applyFix(SplFileInfo $file, Tokens $tokens): void
+    {
+        foreach ($tokens as $index => $token) {
+            if (! $token->isGivenKind(T_DOC_COMMENT)) {
+                continue;
+            }
+
+            if (! $this->isArrayPropertyDocComment($token)) {
+                continue;
+            }
+
+            $semicolonSignOrArrayOpenerPosition = $tokens->getNextMeaningfulToken($index + 4);
+            $semicolonSignOrArrayOpenerToken = $tokens[$semicolonSignOrArrayOpenerPosition];
+            if ($semicolonSignOrArrayOpenerToken->equals('[')) {
+                // token after property equal is start of an array
+                continue;
+            }
+
+            // token after property is ; - its end => so no definition
+            $this->addDefaultValueForArrayProperty($semicolonSignOrArrayOpenerToken);
+        }
+    }
+
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Array property should have default value, to prevent undefined array issues.',
+            []
+        );
+    }
+
+    private function isArrayPropertyDocComment(Token $token): bool
+    {
+        $docBlock = new DocBlock($token->getContent());
+
+        if (count($docBlock->getLines()) === 1) {
+            return false;
+        }
+
+        if (! $docBlock->getAnnotationsOfType('var')) {
+            return false;
+        }
+
+        $varAnnotation = $docBlock->getAnnotationsOfType('var')[0];
+        if (! Strings::contains($varAnnotation->getTypes()[0], '[]')) {
+            return false;
+        }
+
+        return true;
+    }
+
+    private function addDefaultValueForArrayProperty(Token $semicolonToken): void
+    {
+        $semicolonToken->setContent(' = [];');
```

Comment:
This is violation and root cause of your problem reported by Travis.

Basically, `token_get_all` return an array collection of tokens.
All of them are either array (eg `[T_CLASS, 'class', _line_number_]`) or primitive string (eg `;`).
At PHP CS Fixer, we wrap that array collection in `Tokens` class and single token (which we call token prototype) in `Token` (to normalize access across array/string token).

Token must be a valid representation of single token in language parsing terminology.
Here, you try to put 6 tokens in one instance (`space, =, space, [, ], ;`). That's abuse of token representation and that's why it cause you an issue.
Instead of overriding token with a lot of code, you need to inject new tokens into collection.